capa/main.py

```diff
@@ -1028,6 +1030,7 @@ def main(argv=None):
     )
     install_common_args(parser, {"sample", "format", "backend", "signatures", "rules", "tag"})
     parser.add_argument("-j", "--json", action="store_true", help="emit JSON instead of text")
+    parser.add_argument('-o', '--outfile', type=str, help='write Results to .json file')
```

Comment:
we're already a little sensitive to how many cli arguments capa supports, so lets ask that the user pipe the --json output to a file, rather than writing it ourselves:

```console
$ capa --json foo.exe > output.json
```

Reply:
@williballenthin the end goal here is to deserialize the outputs.json file. Which seems far as outputs.json stores the tabulated data. I can only think of splitting the data into sections and parsing. please guide me for this

Reply:
when using the option ` --json` then the output is in a json format (that corresponds to the "ResultDocument" class). you can use something like the following to load the JSON back into a ResultDocument instance:

```py
with open("result.json", "rb") as f:
    buf = f.read()
s = buf.decode("utf-8")
doc = capa.result.ResultDocument.parse_raw(s)
```